BDArmory/Extensions/VesselExtensions.cs

```diff
@@ -89,7 +89,7 @@ public static float GetRadius(this Vessel vessel, Vector3 fireTransform = defaul
 #if DEBUG
                 if (radius < bounds.x / 2f && radius < bounds.y / 2f && radius < bounds.z / 2f) Debug.LogWarning($"DEBUG Radius {radius} of {vessel.vesselName} is less than half its minimum bounds {bounds}");
 #endif
-                return radius;
+                return Mathf.Max(radius, Mathf.Max(Mathf.Max(vessel.vesselSize.x, vessel.vesselSize.y), vessel.vesselSize.z) / 2f); // clamp bounds to vesselsize
```

Comment:
No. That was to prevent it from going above half the largest dimension as an upper limit bound, to fix the issue with VesselMovered (and possibly other cases as well, reproduction was inconsistant) craft being reported with multi-hundred m radii